.github/workflows/continuous-integration.yml

```diff
@@ -28,11 +28,19 @@ jobs:
       run: dotnet build --no-restore --configuration Release
     - name: Test
       run: dotnet test --verbosity normal -p:CollectCoverage=true -p:CoverletOutputFormat=opencover
+    - name: Current directory
+      run: dir
+    - name: Upload Test Report to Codecov
+      uses: codecov/codecov-action@v1
```

Comment:
This is out of date and I believe decommissioned.  Update to @ 3

Reply:
Confirmed that it was deprecated, see readme - https://github.com/codecov/codecov-action

Reply:
The latest I saw from the above link was v2 but I've updated it to v3 anyway.

Reply:
https://github.com/codecov/codecov-action/releases